elm.json

```diff
@@ -4,7 +4,19 @@
   "summary": "Tools for coordinating embedded apps via iframes.",
   "license": "MIT",
   "version": "1.0.0",
-  "exposed-modules": ["HostProgram", "ClientProgram", "ClientMessage"],
+  "exposed-modules": [
+    "Message.AppToClient",
+    "Message.AppToHost",
+    "Message.ClientToApp",
+    "Message.ClientToHost",
+    "Message.HostToApp",
+    "Message.HostToClient",
+    "Message.Navigation",
+    "Message.PubSub",
+    "Message.Toast",
+    "ClientProgram",
+    "HostProgram"
```

Comment:
We need a major version bump if any of these change, since they're part of the API, so we need to expose them. That means the elm package manage will also require that we document them. That's why there are new docs below, which IMO makes the code better as well.